editoast/src/generated_data/error/catenaries.rs

```diff
@@ -59,10 +68,50 @@ pub fn check_catenary_track_ranges(
     infra_errors
 }
 
+/// Checks that catenaries overlapping
+pub fn check_overlapping(infra_cache: &InfraCache, _: &Graph) -> Vec<InfraError> {
+    let mut overlapping_sc = HashSet::new();
+    // Key: (Track, Voltage, Direction)
```

Comment:
I think you don't want to check for overlapping catenaries *of same voltage* but simply for overlapping catenaries, is that right ?

Reply:
I think the tests are failing because of that :grin: